documents/dispute.md

```diff
@@ -0,0 +1,286 @@
+## Document
+
+### Fetch All Disputes
+
+```rb
+Razorpay::Dispute.all()
```

Comment:
Do we allow any params for pagination ?

Reply:
No params according to this doc https://razorpay.com/docs/api/disputes/fetch-all